dbt_subprojects/dex/models/automated_trades/_schema.yml

```diff
@@ -107,14 +101,6 @@ models:
       contributors: 0xBoxer, hosuke
     config:
       tags: [ 'dex' ]
-    data_tests:
-      - dbt_utils.unique_combination_of_columns:
-          combination_of_columns:
-            - blockchain
-            - project
-            - version
-            - tx_hash
-            - evt_index
```

Comment:
why are we removing this, if the spell is still materialized incrementally?
should ideally keep and match unique keys in model

Reply:
not applicable for this model, but will add test to mapped model